app/src/main/res/values/strings.xml

```diff
@@ -113,23 +114,23 @@
     <string name="sharedpreference_sort" translatable="false">sharedpreference_sort</string>
     <string name="sharedpreference_sort_order" translatable="false">sharedpreference_sort_order</string>
     <string name="sharedpreference_sort_direction" translatable="false">sharedpreference_sort_direction</string>
-    <string name="intent_import_card_from_url_share_text">I want to share a card with you</string>
+    <string name="intent_import_card_from_url_share_text">I want to share something from Catima with you</string>
```

Comment:
This is shown to people who may not have the app. "Something from Catima" is super unclear. Like, a card? A movie? A banana?

Reply:
See, it could be anything. Who doesn't want to click that X)